gazebo_ros2_control/package.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>gazebo_ros2_control</name>
+  <version>0.0.1</version>
+  <description>gazebo_ros2_control</description>
+
+  <maintainer email="jrivero@osrfoundation.org">Jose Luis Rivero</maintainer>
```

Comment:
Not sure what @j-rivero thinks. @ahcorde , would you like to be listed as the maintainer? At least for the near future?